app/components/public/side-menu.js

```diff
@@ -13,12 +14,19 @@ export default class SideMenu extends Component {
   @tracked
   showSessions = false;
 
+  async didRender() {
+    super.didRender();
+    const anchor_tag = window.location.hash;
+    $('html, body').animate({
+      scrollTop: $(anchor_tag).offset().top
```

Comment:
Throwing `Cannot read property 'top' of undefined`

Reply:
Looking into this.

Reply:
Is this the only error?

Reply:
Yes, mostly